### PR TITLE
Ticket #2 - Logout 

### PR DESCRIPTION
**Pull request by cresuta**
# Description

This PR is for Ticket #2, which allows the user to logout. The Logout method in the Account Controller was tested and functions properly when the user clicks the "Logout" link in the navbar. The Layout view file in the /Shared directory was checked and both asp-controller and asp-action have correct values 
that allow for a successful logout, where the user is redirected to the Home page.

## Type of change:

- [ ] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# Testing Instructions:

1) If you currently do not have access to this branch, go ahead and run a <git fetch --all>, then <git checkout cr-ticket2> to navigate to this branch to view changes
2) Login with user email of "admin@example.com"
3) Go to the Application tab in DevTools and go to the Cookies dropdown in the side panel view of Storage to access your localhost information
4) If there is a record for ".AspNetCore.Cookies" then we know that a current user successfully logged on
5) Go ahead and click the link in the navbar to Logout
6) Check to see that  ".AspNetCore.Cookies" is removed, which means that the current user successfully logged out

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings or errors
- [x] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by mtgoat** [comment]
- tested, no issues found

**Comment by mtgoat** [comment]
- tested, no issue found

**Comment by Jordan-Twyman** [comment]
tested, works for me

File: TabloidMVC/Controllers/AccountController.cs [hunk 1]
```diff
@@ -50,6 +50,7 @@ public async Task<IActionResult> Login(Credentials credentials)
             return RedirectToAction("Index", "Home");
         }
 
+        // Logout will redirect to home page, where user will have option to log back in
```
Comment (jordan-castelloe) [inline TabloidMVC/Controllers/AccountController.cs:53]: Works as expected! Great job. In general, we probably don't need a full PR for a comment-- typically we'd just add comments as we go, or if we notice a lot of functionality is missing documentation we could add comments for multiple features at once and try to group them all on the same PR. Just a note for next time :)

**Review by jordan-castelloe — APPROVED** [review]
You're good to merge!

**Pull request merged**